ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml

```diff
@@ -103,6 +103,20 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: openshift-e2e-gcp
+- as: e2e-bastion-periodic
+  cluster: build01
+  cron: 0 0 * * *
+  steps:
+    test:
+    - as: e2e-bastion-periodic
+      commands: KUBECONFIG=$SHARED_DIR/kubeconfig make setup-test-cluster && make
```

Comment:
seems like you are setting KUBECONFIG only for the first make command
I think it'd make more sense to use
```
KUBECONFIG=$SHARED_DIR/kubeconfig make functests-on-ci
```

Reply:
good catch. I had that originally iirc, but we split it out into pieces to see at what point it was really failing. I did mess up setting the KUBECONFIG when I split it out though